### ISSUE TITLE
Slider default value doesn't stay inside range

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fyne#1128]
### Describe the bug:
I noticed that having the slider widget inside a grid layout causes the slider to take some sort of weird direction intruding into the grid space assigned to another widget. Trying to drag and tap on the slider makes it go back to the correct spot and functionality. 

### To Reproduce:
Steps to reproduce the behaviour:
1. Create a slider.
2. Place the slider inside a grid layout (2 columns in my case).
3. Start the application and notice how the slider is inside the other space.

### Screenshots:

![slider-bug](https://user-images.githubusercontent.com/25466657/85569975-48d92880-b633-11ea-83de-4064264c6fdc.gif)

### Example code:

```go
package main

import (
	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/layout"
	"fyne.io/fyne/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Slider")

	slider := widget.NewSlider(2, 6)

	content := fyne.NewContainerWithLayout(layout.NewGridLayout(2), widget.NewLabel("Buggy slider ->"), slider)

	w.SetContent(widget.NewVBox(content))
	w.ShowAndRun()
}
```

### Device (please complete the following information):
 - OS: Linux (Solus 4.1)
 - Version: 5.6.18
 - Go version: 1.14.4
 - Fyne version: v1.3.0

**Comment by andydotxyz** [comment]
Ah yes, the default value is not checked to be min <= val <= max. This is an easier fix than I thought :)

**Comment by andydotxyz** [comment]
Fixed on develop for 1.3.1 :)
